core/src/main/java/org/elasticsearch/action/DocWriteResponse.java

```diff
@@ -314,8 +307,8 @@ public XContentBuilder innerToXContent(XContentBuilder builder, Params params) t
                 .field(_ID, id)
                 .field(_VERSION, version)
                 .field(RESULT, getResult().getLowercase());
-        if (getTookInNanos() >= 0) {
-            builder.field(TOOK, getTookInMillis()); // Building 'took' in milliseconds for readability
+        if (getTook().nanos() >= 0) {
+            builder.field(TOOK, getTook().millis());    // Building 'took' in milliseconds for readability
```

Comment:
`took` is the one case we do this in Elasticcsearch. Usually we use `builder.timeValueField` instead. But I believe we don't do that for `took` for historical reasons.